manual/manual_backend.rst

```diff
@@ -34,7 +34,7 @@ Each :py:class:`Backend` object is aware of the restrictions of the underlying d
 .. jupyter-input::
 
     from pytket.extensions.qiskit import IBMQBackend, AerStateBackend
-    dev_b = IBMQBackend("ibmq_london")
+    dev_b = IBMQBackend("ibmq_quito")
```

Comment:
Really we should rerun all these snippets and replace the outputs. But since `ibmq_london` and `ibmq_quito` have the same topology, and numerical results can be expected to differ from run to run even on the same machine, I think it is fine to just switch the name.

Reply:
Yeah this was my thinking in my choice of backend replacement.